draft-ietf-mls-protocol.md

```diff
@@ -1867,6 +2018,43 @@ A member of the group applies a Remove message by taking the following steps:
 
 * Blank the intermediate nodes along the path from the removed leaf to the root
 
+### PSK
+
+A PSK proposal can be used to request that a PSK be injected into the
+key schedule in the process of advancing the epoch.
+
+~~~~~
+struct {
+    PreSharedKeyID psk;
+} PSK;
+~~~~~
```

Comment:
Nit: I would prefer to spell out PreSharedKey.

Do we need a separate structure here, or could we re-use the PreSharedKeys struct defined above?

Reply:
I'll change it such that it spells it out. As I mentioned in my comment above, the Proposal only contains a single `PreSharedKeyID`, while `PreSharedKeys` contains an array. We could change that, though. I don't have a strong opinion on this, so I'll leave it as it is for now.

Reply:
Terminology is a bit tricky. We have "PSK"s (i.e. the concrete keys), but now we also have "PreSharedKey" which is the name of the Proposal. However, the Proposal in turn only contains `PreSharedKeyID`s, i.e. pointers to how the PSK can be derived.